samples/cpp/speech_sample/main.cpp

```diff
@@ -232,6 +252,8 @@ int main(int argc, char* argv[]) {
                 throw std::runtime_error("Cannot open model file " + FLAGS_rg);
             }
             executableNet = core.import_model(streamrq, deviceStr, genericPluginConfig);
+            // loading batch from exported model
+            batchSize = (uint32_t)executableNet.input(0).get_shape()[0];
```

Comment:
Why batch size is in zero dimension always for exported model?

Reply:
ok, I will correct